src/pages/index.js

```diff
@@ -59,26 +135,58 @@ const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+const profileAvatar = document.querySelector("#profile-image-id");
 const profileEditForm = document.forms["edit-profile-modal__form"];
 const addCardFormElement = document.forms["add-card-modal__form"];
-const cardlistEl = document.querySelector("card-list-id");
+const avatarModalForm = document.forms["edit-avatar-modal__form"];
 
 const userInfo = new UserInfo({
   profileTitle,
   profileDescription,
+  profileAvatar,
+});
+
+const avatarModal = new PopupWithForm({
+  popupSelector: "#edit-avatar-modal",
+  handleFormSubmit: (inputValue) => {
+    console.log(inputValue);
+    avatarModal.setLoading(true, "Saving");
+    api
+      .avatarModal(inputValue)
+      .then((info) => {
+        userInfo.setUserInfo(info);
+        avatarModalValidator.disableButton();
```

Comment:
* The modals don't close after submission in `.then`. 


Please, close the modal after this line